swagger/v1-prerelease/swagger.yaml

```diff
@@ -3219,18 +3219,15 @@ components:
         - canceled
         - succeeded
       type: string
-    PartitionEntry:
+    Partition:
       type: object
       properties:
-        partitionKey:
+        key:
           type: integer
           format: int32
-        partitionName:
+        name:
           type: string
           nullable: true
-        createdDate:
```

Comment:
Yeah.... we'll need to add this back lol

Reply:
Oh dang, all of the names will need to be updated too, but I think we can isolate these changes to this code path